pyspark_ai/react_spark_sql_agent.py

```diff
@@ -26,7 +26,10 @@ def _agent_type(self) -> str:
     @classmethod
     def create_prompt(cls, tools: Sequence[BaseTool]) -> BasePromptTemplate:
         """Return default prompt."""
-        return SPARK_SQL_PROMPT
+        if len(tools) == 2:
```

Comment:
Shall we can whether SimilarValueTool in the tools instead? In the future there can be more tools